suse_caasp

```diff
@@ -331,6 +331,21 @@ find_and_log_files () {
     fi
 }
 
+#############################################################
+section_header "Checking transactional-update service ..."
+OF=transactional-update.txt
+plugin_message "All data stored in $OF"
+
+if check_rpm transactional-update; then
+    rpm_verify $OF transactional-update
+    log_cmd $OF 'systemctl status transactional-update.service'
+    log_cmd $OF 'systemctl status transactional-update.timer'
+    log_cmd $OF 'journalctl -u transactional-update'
+    log_cmd $OF 'snapper list'
+    find_and_log_files $OF /var/log/transactional-update.log
```

Comment:
This doesn't seem to actually find the logfile - even though it exists

Reply:
Fixed - find_and_log_files seems to expect a directory, `log_files` seems to do whats needed.

Reply:
Yeah, it works only with directories